.travis.yml

```diff
@@ -41,6 +39,8 @@ install:
 
 script:
   - make test
+  - sudo apt-get install python-pip
+  - sudo pip install pylint
   - jdk_switcher use oraclejdk8
   - |-
     PYTHONPATH=libexec/python/ pylint libexec/python/ -r n --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" > pylint-report.txt
```

Comment:
For example why not like:

      PYTHONPATH=libexec/python/; pylint libexec/python/ -r n --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" > pylint-report.txt" or maybe try just going there like:

      cd libexec/python && pylint  -r n --msg-template="{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}" > pylint-report.txt"